Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -324,6 +324,8 @@ namespace MultiplayerSample
                 }
             }
 
+            // Print the player results to server.log for tracking tournament winners.
+            AZ_Info("NetworkMatchComponentController", "Match Results: Player %s score %i, armor %i.", state.m_playerName.c_str(), state.m_score, state.m_remainingArmor);
```

Comment:
Nit: %i -> %u, these are unsigned ints.

Reply:
Suggestion: If you use AZLOG(category, ...) then the printing could be off by default, but could be enabled on the server with a command-line argument.

Reply:
Oh, would be good to add that cli as a comment in ILogger.h. Dont suppose you know the cvar off-hand?

Reply:
> Would love to see sample of logs in situ.

+1

Reply:
![image](https://user-images.githubusercontent.com/32776221/228972632-29601f86-02d8-4268-a2eb-de51ce691946.png)

Reply:
Updated to pretty print with a nice format and sorted
![image](https://user-images.githubusercontent.com/32776221/228978158-4cf18276-21b1-4bc3-9f87-fbb374fb9051.png)